go.mod

```diff
@@ -3,20 +3,30 @@ module github.com/terraform-providers/terraform-provider-external
 go 1.18
 
 require (
-	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
+	github.com/hashicorp/terraform-plugin-docs v0.13.0
```

Comment:
**drive-by reminder:** we should avoid this dependency from upgrading to `v0.14.0` as it'll update `terraform-exec` to `v0.18.0` which will break any acceptance tests that use `TestStep > ExpectError`. We can probably just let dependabot handle the upgrade when we release `terraform-plugin-docs` [v0.14.1](https://github.com/hashicorp/terraform-plugin-docs/milestone/9)